docs/startup/releases/versioned/release-notes.md

```diff
@@ -42,8 +43,12 @@ title: Release Notes
 
 ## Release Notes
 <!-- Add notes on new features, modified behavior of existing features, operational/performance improvements, new tools etc -->
-
-<TBD>
+* [SAMZA-1510](https://issues.apache.org/jira/browse/SAMZA-1510) - Samza SQL
+* [SAMZA-1438](https://issues.apache.org/jira/browse/SAMZA-1438) - Event-hub connectors for Samza
+* [SAMZA-1421](https://issues.apache.org/jira/browse/SAMZA-1421) - Add support for durable state for high level API
+* [SAMZA-1392](https://issues.apache.org/jira/browse/SAMZA-1392) - KafkaSystemProducer performance and correctness with concurrent sends and flushes
```

Comment:
s/KafkaSystemProducer performance and correctness with concurrent sends and flushes/"Performance and correctness fixes for KafkaSystemProducer for concurrent sends and flushes"